apps/assessment_registry/filter_set.py

```diff
@@ -0,0 +1,11 @@
+from user_resource.filters import UserResourceGqlFilterSet
+from utils.graphene.filters import IDListFilter
+
+
+class AssessmentRegistryGQFilterSet(UserResourceGqlFilterSet):
+    ids = IDListFilter(method='filter_assessment_registry_id', help_text='Empty ids are ignored')
+
+    def filter_assessment_registry_id(self, qs, _, value):
```

Comment:
Let's use apps/ary/filters.py as a base to define other additional fields to add.